DDG4/python/DDSim/DD4hepSimulation.py

```diff
@@ -628,7 +628,9 @@ def __printSteeringFile(self, parser):
         else:
           steeringFileBase += "SIM.%s = %s" % (parName, str(parameter))
         steeringFileBase += "\n"
-    logger.info("%s", steeringFileBase)
+    for line in steeringFileBase.splitlines():
+      print(line)
```

Comment:
You have to add `from __future__ import print_function` since there is no other usage of `print` in this file, the import is not added at the beginning.

Reply:
added the import